Makefile.codegen

```diff
@@ -43,3 +52,15 @@ generate-clean: clean
 	rm -f $(GOPATH)/bin/lister-gen
 	rm -f $(GOPATH)/bin/openapi-ge
 	rm -f $(GOPATH)/bin/gen-apidocs
+
+# Verifies that generated code is in sync with implementation
+verify-generation-complete: generate
```

Comment:
@garethjevans fyi, this should be all that's needed to run and verify that all generated resources are up to date